tearup.yaml

```diff
@@ -640,7 +640,17 @@
         state: present
         reload: yes
 
-    - name: disable send_redirects
+    - name: disable net.ipv4.conf.all.send_redirects
+      sysctl:
+        # eth1 is the gateway interface for leaf hosts
```

Comment:
The comment does not apply. Perhaps we should explain why we need to disable for `all` and for `eth1` at the same time.

Reply:
done